### ISSUE TITLE
Extra fields in response when getting access token causing crash

### ISSUE THREAD

**Issue by marthinsen** [CamiloTerevinto/TerevintoSoftware.PythonPkceClient#2]
The OAuth provider i'm using is adding extra fields to to the response when getting the access token, namely `remember` and `issues_at`, I guess they are non-standard.
```
{
  'access_token': '...'
  'remember': False
  'refresh_token': '...'
  'scope': 'openid profile'
  'id_token': '...'
  'token_type': 'Bearer'
  'expires_in': 3600
  'issued_at': 1680725436
}
```
I have "solved it" locally by adding `**kwargs` as the last argument of the `__init__` function of the `PkceToken` class...

**Comment by CamiloTerevinto** [comment]
Hey @marthinsen, thanks for the bug report! I have just fixed this and released 0.0.3: https://pypi.org/project/TerevintoSoftware.PkceClient/0.0.3/

I'll leave this open so you can confirm if it's working for you.

**Comment by marthinsen** [comment]
It works like a charm. Thanks for the quick response!
